TESTING-INSTRUCTIONS.md

```diff
@@ -2,6 +2,107 @@
 
 ## Unreleased
 
+### Payment gateway suggestions feature
+
+1. Note that the `payment-gateway-suggestion` feature should be turned on in your current build
+2. Navigate to the homescreen via WooCommerce -> Home
+3. Click on "Set up payments"
+
+#### How to test
+
+Individual payment gateway plugins dictate the settings and connection flow.  For testing purposes, we'll test both the default behavior of the gateway and the enhanced configuration behavior.
+
+1. On the payments task, click "Set up" or "Enable" next to a gateway
+2. Note that the gateway is installed if it requires a plugin
+3. On the connection step, a button should be presented that links to the gateway's (legacy) settings screen
+4. Delete the plugin
+5. Clone and checkout one of the gateway PRs in this issue - https://github.com/woocommerce/woocommerce-admin/issues/6878
```

Comment:
I am not sure if this is do-able by GlobalStep testing, maybe we want to provide plugin zip's of each PR mentioned in #6878 (we could just upload them to the PR thread)?

Reply:
That's a good idea.  I'll update this and make an issue to collect those plugins.